src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,38 @@
+@import "../utils/variables";
+
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, $catalog-card-width);
+  gap: 48px;
+
+  max-width: 944px;
+  margin: 0 auto;
```

Comment:
Don't add external styles (positioning or margins) to BEM-blocks. Use mix where necessary and move all external styles under element selector.

Reply:
I've created an element selector (wrapper), and have moved it to a separate file and separate folder. Though, not sure whether it is in line with BEM...